tests/e2e-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"type": "module",
+	"name": "@sap-ai-sdk/e2e-tests",
+	"version": "0.0.1",
+	"main": "./dist/index.js",
+	"types": "./dist/index.d.ts",
+	"files": [
+		"dist/**/*.js",
+		"dist/**/*.js.map",
+		"dist/**/*.d.ts",
+		"dist/**/*.d.ts.map"
+	],
+	"dependencies": {
+		"@sap-ai-sdk/sample-code": "workspace:^"
+	},
+	"scripts": {
+		"compile": "tsc",
+		"e2e-test": "NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
Among other required settings, this seems to be required, because [jest support for ESM is experimental](https://jestjs.io/docs/ecmascript-modules). See also [this SO question](https://stackoverflow.com/a/75001235).

However, this means tests don't work if triggered via an IDE. One has to configure their jest runner to pass this node option.